patterns/blueprint-vpc-lattice/cluster2/postgres-setup/README.md

```diff
@@ -0,0 +1,6 @@
+## Setting up PostgreSQL database
```

Comment:
Lets remove all of the database dependencies unless there is a strong reason why this needs to be included in the pattern

Reply:
I got this idea from a blog post that's already linked to this problem/request which depends upon the aurora RDS . 
https://aws.amazon.com/blogs/containers/application-networking-with-amazon-vpc-lattice-and-amazon-eks/ 

i am happy to work on a different scenario if you could direct me in the right direction